rules/prefer-textcontent.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const getQuotedReplacement = (node, value) => {
+	const leftQuote = node.raw.charAt(0);
+	const rightQuote = node.raw.charAt(node.raw.length - 1);
+	return `${leftQuote}${value}${rightQuote}`;
+};
+
+const message = 'Prefer `textContent` over `innerText`.';
+
+const create = context => {
+	return {
+		MemberExpression: node => {
+			const {property} = node;
+
+			if (property.type === 'Identifier' && !node.computed && property.name === 'innerText') {
+				// Handles node.innerText
+				context.report({
+					node,
+					message,
+					fix: fixer => fixer.replaceText(property, 'textContent')
+				});
+			} else if (property.type === 'Literal' && property.value === 'innerText') {
```

Comment:
Could've used [`astUtils.getPropertyName(node)`](https://github.com/jfmengels/eslint-ast-utils#astutilsgetpropertynamenode) while also fixing `node["innerText"]` into `node.textContent`

Reply:
good point. I wasn't sure if we want to retain the existing syntax when autofixing.

Reply:
I'm not sure either, @sindresorhus, what do you think?

Reply:
My take would be that I don't care, since property access variant is never seen in the wild. The rule might as well ignore it completely.

Reply:
Yeah, we can just ignore it.